app/src/test/java/org/simple/clinic/contactpatient/CallPatientUiRendererTest.kt

```diff
@@ -371,10 +372,44 @@ class CallPatientUiRendererTest {
     verifyNoMoreInteractions(ui)
   }
 
+  @Test
+  fun `display registered at facility label text when patient's registered facility is the same as user's current facility`() {
```

Comment:
As far as I remember, we need to show the "Registered label" when the assigned facility ID is same as the current facility ID. If not, then we show transferred from label. In both scenarios we show the registered facility name, just the label changes.

Reply:
Yes this update branch only changes the label text name